BRTesterMac/src/app.re

```diff
@@ -40,9 +40,15 @@ let render = element => {
     window#setTitle(appName);
     window#setContentView(root.view);
 
-    window#windowDidResize(_ => React.RunLoop.performLayout(root));
+    window#windowDidResize(_ =>
+      React.RunLoop.performLayout(~height=window#contentHeight, root)
```

Comment:
I think we should make `setWindowHeight` separate from `performLayout` and always pass `height` to `performLayout`.